tests/integration/test_store.py

```diff
@@ -906,6 +909,9 @@ def counted_check_challenge(*args, **kwargs):
     auth_count_3 = counter[0]
     assert auth_count_2 < auth_count_3
 
+    # Give Relay some time to process the auth response and mark itself as ready
+    sleep(0.1)
```

Comment:
Ran this 10 times on a different branch, `0.1` seems to work fine:

https://github.com/getsentry/relay/actions/runs/3318797256/jobs/5483154174#step:10:662